src/ensembl/src/content/app/entity-viewer/shared/components/entity-viewer-sidebar/entity-viewer-sidebar-modal/modal-views/EntityViewerSidebarShare.tsx

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
Do we even need this component (as well as the PersonalData component) now? They are definitely not for MAP, and probably not for MVP either.

Reply:
Those are just placeholders for the future. This is not used anywhere in the code, also we now have lazyload. Do we really need to remove this?

Reply:
decided to remove unused codes

Reply:
Erm... this file is still in the diff.